bay/images/nginx/helpers/200_x-sdp-app-type-header.conf

```
@@ -0,0 +1 @@
+add_header X-Sdp-App-Type "tide";
```

We might be able to do this with environment variable substitution. The base Amazee nginx container runs `ep` over the helpers directory during the entry of the container.

`ep` (envplate) supports environment variable substitution in static files.

```suggestion
add_header X-Sdp-App-Type "${SDP_APP_TYPE:-tide}";
```

---

https://github.com/amazeeio/lagoon/blob/master/images/nginx/docker-entrypoint#L22